subiquity/tests/test_timezonecontroller.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2021 Canonical, Ltd.
```

Comment:
Tests should be close to implementation imo, so this should be in subiquity/server/controllers/tests

Reply:
#986